### PR TITLE
Added page to build list.

### PR DESCRIPTION
**Pull request by Raffo**
This adds a `--page=N` flag to get the old builds from the CLI as commented in https://github.com/drone/drone/issues/1494#issuecomment-436556751 

Requires https://github.com/drone/drone-go/pull/39 to be able to this to work (needs vendoring).

### CODE REVIEW

**Comment by Raffo** [comment]
Clearly this will pass only after the tests are merged. Is there any way to add tests to verify this functionality?

File: drone/build/build_list.go [hunk 1]
```diff
@@ -36,6 +36,11 @@ var buildListCmd = cli.Command{
 			Usage: "limit the list size",
 			Value: 25,
 		},
+		cli.IntFlag{
+			Name:  "page",
+			Usage: "size of the page",
```
[the remark this answers is not in the document]
Comment (Raffo) [reply drone/build/build_list.go]: Correct, will edit that.

**Comment by Raffo** [comment]
@tboerger What is the command that you usually use to update the dependencies? `dep ensure -update github.com/drone/drone-go` changes quite a lot of stuff in the Gopkg.lock, is there anything you can recommend to deal with this?

**Comment by tboerger** [comment]
> @tboerger What is the command that you usually use to update the dependencies? `dep ensure -update github.com/drone/drone-go` changes quite a lot of stuff in the Gopkg.lock, is there anything you can recommend to deal with this?

That's the way to go, it changes more within the file if you are using a more current version of dep because it adds more or updated information to the lock file. IMHO this should be fine.

**Comment by Raffo** [comment]
@tboerger okay, thank you, I updated it and the tests are now green.

**Comment by Raffo** [comment]
@bradrydzewski I think this can be merged now 😄

**Pull request merged**